2023-01-13T09:59:53.464017+0100	| INFO	| run_mission.py:run:43 	- Waiting for drone to connect...>
2023-01-13T09:59:53.476374+0100	| INFO	| run_mission.py:run:46 	- Connected to drone!>
2023-01-13T09:59:53.477201+0100	| INFO	| run_mission.py:run:49 	- Waiting for drone to have a global position estimate...>
2023-01-13T10:00:06.085162+0100	| INFO	| run_mission.py:run:52 	- Global position estimate OK>
2023-01-13T10:00:06.111196+0100	| INFO	| run_mission.py:run:60 	- Arming>
2023-01-13T10:00:06.320433+0100	| INFO	| run_mission.py:run:97 	- Uploading mission 0>
2023-01-13T10:00:06.322803+0100	| DEBUG	| run_mission.py:monitor_distance:159 	- Distance monitoring enabled>
2023-01-13T10:00:06.325318+0100	| DEBUG	| run_mission.py:do_mast_recognition:173 	- Mast recognition called>
2023-01-13T10:00:06.325594+0100	| DEBUG	| run_mission.py:do_mast_recognition:174 	- Has Found Mast: False>
2023-01-13T10:00:06.326130+0100	| INFO	| run_mission.py:do_mast_recognition:178 	- Taking image>
2023-01-13T10:00:06.374133+0100	| DEBUG	| run_mission.py:image_contains_mast:206 	- image_contains_mast called>
2023-01-13T10:00:07.352596+0100	| DEBUG	| run_mission.py:image_contains_mast:212 	- image_contains_mast returned False, confidence: 0.5724779367446899>
2023-01-13T10:00:07.353196+0100	| DEBUG	| run_mission.py:do_mast_recognition:192 	- Mast-check completed.>
2023-01-13T10:00:07.354228+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:07.356608+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:07.359795+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:07.363192+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:07.615251+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:07.619545+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:07.886473+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:07.889104+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:08.077079+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:08.079771+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:08.355782+0100	| INFO	| run_mission.py:run:102 	- Starting mission>
2023-01-13T10:00:08.408080+0100	| INFO	| run_mission.py:run:105 	- Now checking mast F2542G at 10.554784917159, 55.3743128282325>
2023-01-13T10:00:08.409315+0100	| INFO	| run_mission.py:run:113 	- Clearing current mission>
2023-01-13T10:00:08.530169+0100	| INFO	| run_mission.py:run:117 	- Returning to base>
2023-01-13T10:00:08.673374+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:08.675663+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:09.249462+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:09.252432+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:09.651733+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:09.655374+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:09.843792+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:09.847396+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:10.126297+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:10.128716+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:10.602964+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:10.605988+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:10.908000+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:10.911354+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:10.982642+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:10.986401+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:11.356938+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:11.359836+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:11.654895+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 411.0]>
2023-01-13T10:00:11.659137+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:12.133284+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:12.137051+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:12.355089+0100	| DEBUG	| run_mission.py:do_mast_recognition:195 	- Mast recognition disabled while returning.>
2023-01-13T10:00:12.409385+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:12.412253+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:12.695867+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:12.698756+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:12.787950+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:12.790788+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:13.166935+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:13.170675+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:13.641233+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:13.643720+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:13.920421+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:13.922465+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:14.094124+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:14.097046+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:14.371557+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:14.373876+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:14.572118+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:14.575000+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:14.955209+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:14.957863+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:15.224545+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:15.227777+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:15.503549+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:15.505868+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:15.790650+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:15.793649+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:15.953171+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:15.955751+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:16.228024+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:16.230552+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:16.908221+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 423.0]>
2023-01-13T10:00:16.911153+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:17.185307+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 412.0]>
2023-01-13T10:00:17.187786+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:17.357111+0100	| DEBUG	| run_mission.py:do_mast_recognition:195 	- Mast recognition disabled while returning.>
2023-01-13T10:00:17.471262+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 414.0]>
2023-01-13T10:00:17.473568+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:17.749367+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 417.0]>
2023-01-13T10:00:17.752932+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:18.126785+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 416.0]>
2023-01-13T10:00:18.128880+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:18.404668+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 417.0]>
2023-01-13T10:00:18.407083+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:18.887738+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 427.0]>
2023-01-13T10:00:18.891093+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:19.563707+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 427.0]>
2023-01-13T10:00:19.566226+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:20.028693+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 417.0]>
2023-01-13T10:00:20.030846+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:20.324354+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 421.0]>
2023-01-13T10:00:20.327238+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:20.533647+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 425.0]>
2023-01-13T10:00:20.536111+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:21.008030+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 431.0]>
2023-01-13T10:00:21.010275+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:21.480960+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 433.0]>
2023-01-13T10:00:21.483440+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:21.751207+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 439.0]>
2023-01-13T10:00:21.753409+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
2023-01-13T10:00:22.135698+0100	| DEBUG	| run_mission.py:monitor_distance:161 	- Distance from sensor: DistanceSensor: [minimum_distance_m: 1.0, maximum_distance_m: 500.0, current_distance_m: 446.0]>
2023-01-13T10:00:22.139290+0100	| INFO	| run_mission.py:monitor_distance:167 	- Obstacle identified, line of sight cannot be confirmed.>
